src/data/thunks.js

```diff
@@ -123,6 +123,19 @@ export function startProctoringExam() {
   };
 }
 
+export function skipProctoringExam() {
+  return async (dispatch, getState) => {
+    const { exam } = getState().examState;
+    if (!exam.id) {
+      logError('Failed to skip proctored exam. No exam id.');
+      return;
+    }
+    await updateAttemptAfter(
```

Comment:
not directly related to this PR but leaving as a general comment since I just noticed. The fact that one of it's arguments is named `promise` (and it resolves to a variable called `data`) makes it a bit non-obvious what's going on when reading through. That function is pretty core it could probably benefit from a docstring and maybe some more explicit variable naming (promise/data)

Reply:
addressed in 6f003f9, let me know what you think

Reply:
great thanks, makes more sense now